src/components/date-picker/picker.vue

```diff
@@ -76,7 +76,11 @@
                         @on-pick-click="disableClickOutSide = true"
                         @get-date="getDate"
                         @on-selection-mode-change="onSelectionModeChange"
-                    ></component>
+                    >
+                        <template #dropFooter v-if="$slots['dropFooter'] || $scopedSlots['dropFooter']">
+                            <slot name="dropFooter"></slot>
+                        </template>
```

Comment:
这里感觉不用这层template了，直接slot就好
```suggestion -2+0
                        <slot name="dropFooter"></slot>
```